index.html

```diff
@@ -0,0 +1,383 @@
+              />
+            </a>
+          </div>
+        </div>
+      </div>
+      <div class="work-card">
+        <img src="./assets/img/laptop 2.png" alt="laptop" />
+        <div class="work-card-text">
+          <h3>Project name goes here</h3>
+          <div class="lang-container">
+            <div>
+              <h4 id="lang">HTML/CSS</h4>
+            </div>
+            <div>
+              <h4 id="lang">Ruby on Rails</h4>
+            </div>
+            <div>
+              <h4 id="lang">JavaScript</h4>
+            </div>
+          </div>
+          <div>
+            <a href="#" class="primary-btn-2"
+              >See this project
+              <img
+                src="./assets/img/right_arrow.png"
+                alt="right_arrow"
+                class="right_arrow"
+              />
+            </a>
+          </div>
+        </div>
+      </div>
+      <div class="work-card">
+        <img src="./assets/img/laptop 3.png" alt="laptop" />
+        <div class="work-card-text">
+          <h3>Project name goes here</h3>
+          <div class="lang-container">
+            <div>
+              <h4 id="lang">HTML/CSS</h4>
+            </div>
+            <div>
+              <h4 id="lang">Ruby on Rails</h4>
+            </div>
+            <div>
+              <h4 id="lang">JavaScript</h4>
+            </div>
+          </div>
+          <div>
+            <a href="#" class="primary-btn-2"
+              >See this project
+              <img
+                src="./assets/img/right_arrow.png"
+                alt="right_arrow"
+                class="right_arrow"
+              />
+            </a>
+          </div>
+        </div>
+      </div>
+      <div class="work-card">
+        <img src="./assets/img/laptop 4.png" alt="laptop" />
+        <div class="work-card-text">
+          <h3>Project name goes here</h3>
+          <div class="lang-container">
+            <div>
+              <h4 id="lang">HTML/CSS</h4>
+            </div>
+            <div>
+              <h4 id="lang">Ruby on Rails</h4>
+            </div>
+            <div>
+              <h4 id="lang">JavaScript</h4>
+            </div>
+          </div>
+          <div>
+            <a href="#" class="primary-btn-2"
+              >See this project
+              <img
+                src="./assets/img/right_arrow.png"
+                alt="right_arrow"
+                class="right_arrow"
+              />
+            </a>
+          </div>
+        </div>
+      </div>
+      <div class="work-card">
+        <img src="./assets/img/laptop 5.png" alt="laptop" />
+        <div class="work-card-text">
+          <h3>Project name goes here</h3>
+          <div class="lang-container">
+            <div>
+              <h4 id="lang">HTML/CSS</h4>
+            </div>
+            <div>
+              <h4 id="lang">Ruby on Rails</h4>
+            </div>
+            <div>
+              <h4 id="lang">JavaScript</h4>
+            </div>
```

Comment:
- [ ] Kindly wrap the technologies in `ul` tag 👍 and each of them a `li`. This way will be semantically correct and the screen readers